tests/lib/util.js

```diff
@@ -158,6 +158,38 @@ describe("util", function() {
             assert.deepEqual(code[1], { rules: { "no-mixed-requires": [1, {"err": ["error", "e"]}] }});
         });
 
+        it("should combine configs and override rules options without array or object", function() {
+
+            var code = [
+                { rules: { "no-mixed-requires": [1, "nconf", "underscore"] } },
+                { rules: { "no-mixed-requires": [2, "requirejs"] } }
+            ];
+
+            var result = util.mergeConfigs(code[0], code[1]);
+
+            assert.strictEqual(result.rules["no-mixed-requires"][0], 2);
+            assert.strictEqual(result.rules["no-mixed-requires"][1], "requirejs");
+            assert.isUndefined(result.rules["no-mixed-requires"][2]);
+            assert.deepEqual(code[0], { rules: { "no-mixed-requires": [1, "nconf", "underscore"] }});
```

Comment:
unclear on the need for these assertions. are you just proving the originals were unmodified?